Week2/homework/map_filter.js

```diff
@@ -3,3 +3,5 @@
 const numbers = [1, 2, 3, 4];
 
 // Add your code here
+const newNumbers = numbers.filter(number => (number % 2 !== 0)).map(num => num * 2);
```

Comment:
Well done here!

As a minor comment, to improve readability you might want to reformat the code a bit:

```js
const newNumbers = numbers
  .filter(number => number % 2 !== 0)
  .map(number => number * 2);
```

Note that you do not need to have parentheses around the filter condition, although it is perfectly fine if it makes it clearer to you.

It would be more consistent if you use the name `number` in both functions.